tests/mock_crust/churn.rs

```diff
@@ -210,9 +259,26 @@ fn churn() {
         // expected_gets.send_and_expect(data_id, auth_s0, auth_g0, &nodes, min_group_size);
         // expected_gets.send_and_expect(data_id, auth_s0, auth_n0, &nodes, min_group_size);
 
-        poll_and_resend(&mut nodes, &mut []);
+        // Test messages from a client to a group and a section...
+        let data_id1 = DataIdentifier::Immutable(rng.gen());
+        let auth_g2 = Authority::ClientManager(rng.gen());
+        let auth_s1 = Authority::Section(section_name);
```

Comment:
Why create a new data ID and section authority? Let's just reuse the ones we already have.
And why does this use a `ClientManager` and not a `Client`?